Packs/Cyberwatch/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### Cyberwatch
+- Changed fetch assets and fetch one asset commmand to work on different routes.
+- Add commands to get and update declarative data on cyberwatch.
```

Comment:
but the fetch assets commands are new commands, so nothing is changed here, just added.
the RN should looks something like this:
 - Added <number> commands:
  - ***command-one***: <description>
  - ***command-two***: <description>